### PR TITLE
Created an empty txt file

### PR DESCRIPTION
**Pull request by guillaume-boutin**
I just created an empty file so i can write a lovely poem.

### CODE REVIEW

**Comment by guillaume-boutin** [comment]
The file is currently empty, but i will soon fill it!

**Comment by guillaume-boutin** [comment]
I'm getting and storing all the webhooks so far!

**Comment by ziad-saab** [comment]
hello world!

File: poem.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Roses are red,
+Violets are blue (sort of),
+But oh!
```
Comment (ziad-saab) [inline poem.txt]: hello

**Comment by ziad-saab** [comment]
xxx

**Pull request closed without merging**